wasm/jit/jit_arm64.go

```diff
@@ -368,7 +368,7 @@ func (c *arm64Compiler) returnFunction() error {
 	// At this point, we have
 	//
 	//      [......., ra.caller, rb.caller, rc.caller, _, ra.current, rb.current, rc.current, _, ...]  <- call frame stack's data region (somewhere in the memory)
-	//                                                 ^
+	//                                                  ^
```

Comment:
actually this should be on comma as this points to the address of a struct, not any specific field.